From 3fff225de35833e62625a75259c610cfa395d953 Mon Sep 17 00:00:00 2001
From: Tomeu Vizoso <tomeu.vizoso@collabora.com>
Date: Thu, 2 Apr 2020 06:50:32 +0200
Subject: [PATCH 011/105] HACK: drm/panfrost: comment reading of
 GPU_LATEST_FLUSH_ID as it causes an external abort

---
 drivers/gpu/drm/panfrost/panfrost_gpu.c | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/drivers/gpu/drm/panfrost/panfrost_gpu.c b/drivers/gpu/drm/panfrost/panfrost_gpu.c
index f1fe70f4042c..d53127a99ee4 100644
--- a/drivers/gpu/drm/panfrost/panfrost_gpu.c
+++ b/drivers/gpu/drm/panfrost/panfrost_gpu.c
@@ -379,7 +379,7 @@ void panfrost_gpu_fini(struct panfrost_device *pfdev)
 
 u32 panfrost_gpu_get_latest_flush_id(struct panfrost_device *pfdev)
 {
-	if (panfrost_has_hw_feature(pfdev, HW_FEATURE_FLUSH_REDUCTION))
-		return gpu_read(pfdev, GPU_LATEST_FLUSH_ID);
+	//if (panfrost_has_hw_feature(pfdev, HW_FEATURE_FLUSH_REDUCTION))
+	//	return gpu_read(pfdev, GPU_LATEST_FLUSH_ID);
 	return 0;
 }
-- 
2.25.0

